src/XCCDF/xccdf_session.c

```diff
@@ -425,10 +425,19 @@ bool xccdf_session_set_profile_id(struct xccdf_session *session, const char *pro
 	return true;
 }
 
-bool xccdf_session_set_profile_id_by_suffix(struct xccdf_session *session, const char *profile_suffix)
+// 0 = no error, 1 = no matches, 2 = multiple matches
+int xccdf_session_set_profile_id_by_suffix(struct xccdf_session *session, const char *profile_suffix)
 {
 	const char *full_profile_id = NULL;
 	struct xccdf_benchmark *bench = xccdf_policy_model_get_benchmark(session->xccdf.policy_model);
+	bool multiple = false;
+
+	const char full_profile_id_prefix[] = "xccdf_org.ssgproject.content_profile_";
```

Comment:
I'm getting segmentation fault when I try it with pointers, I think because I am appending another string to it. Not sure